frontend/src/commons/SoccerTeam.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import styled from 'styled-components/macro';
+
+export default function SoccerTeam({ soccerTeam }) {
+  return <SoccerTeamStyled>{soccerTeam.name}</SoccerTeamStyled>;
+}
+
+const SoccerTeamStyled = styled.span`
+  height: 50px;
+  display: flex;
+  align-items: center;
+  border-radius: var(--border-radius);
+  transition: background var(--speed);
+  padding: 0.2rem;
+  color: var(--blue-50);
+  font-size: var(--size-l);
+  text-decoration: none;
+  margin-right: 2px;
```

Comment:
If you want to string components together, better use `display:grid` on the parent container and space them with `grid-gap`